src/components/header/header.jsx

```diff
@@ -5,6 +5,7 @@ import {
 } from '@material-ui/core';
 import { withRouter } from "react-router-dom";
 import { colors } from '../../theme'
+import ENS from 'ethjs-ens';
```

Comment:
Web3 does not support reverse ENS resolution. Recommended library per official ens docs is ethjs-ens (https://docs.ens.domains/dapp-developer-guide/resolving-names#reverse-resolution)